src/main/java/hu/dpc/phee/operator/entity/outboundmessages/OutboudMessages.java

```diff
@@ -0,0 +1,174 @@
+package hu.dpc.phee.operator.entity.outboundmessages;
+
+import hu.dpc.phee.operator.entity.parent.AbstractPersistableCustom;
+
+import javax.persistence.*;
+
+import hu.dpc.phee.operator.entity.transfer.TransferStatus;
+import hu.dpc.phee.operator.util.SmsMessageStatusType;
+import java.util.Date;
+
+@Entity
+@Table(name = "m_outbound_messages")
```

Comment:
where is the migration script for this table? are we pushing this info to the message gateway DB or the tenant DB?

could not find the DB connection details to the message gateway DB

Reply:
what is the resolution ?

Reply:
we pushing this info to the message gateway DB? or tenant db?

Reply:
message gateway db